lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2518,6 +2518,7 @@ private static function get_block_nodes( $theme_json, $selectors = array() ) {
 	 * Gets the CSS rules for a particular block from theme.json.
 	 *
 	 * @since 6.1.0
+	 * @since 6.6.0 Setting a min-height of HTML for background gradient or background size.
```

Comment:
Tiny nit: we're outputting when a background image is set, so should this be the following:?

```suggestion
	 * @since 6.6.0 Setting a min-height of HTML for background gradient or background image.
```

Reply:
Whoops, forgot about those. Thanks!